src/index.js

```diff
@@ -183,6 +183,13 @@ export default class RNPickerSelect extends PureComponent {
     }
 
     togglePicker(animate = false) {
+        if (this.state.showPicker) {
+            this.props.onBlur()
+        } else {
+            this.props.onFocus()
+        }
+
```

Comment:
extra white space :trollface:

Reply:
this guy

Reply:
making me look baaaaaaad. But good catch thank you ;)

Reply:
👍 🥇